string ssid
---
bool res
string message
